package/post-install.sh

```diff
@@ -11,19 +11,30 @@ CUSTOM_INI_FILE_NAME='ddtrace-custom.ini'
 
 PATH="${PATH}:/usr/local/bin"
 
-if [[ -z "$DD_TRACE_PHP_BIN" ]]; then
-    DD_TRACE_PHP_BIN=$(command -v php)
+# We attempt in this order the following binary names:
+#    1. php
+#    2. php7 (some alpine versions install php 7.x from main repo to this binary)
+#    3. php5 (some alpine versions install php 5.x from main repo to this binary)
+if [ -z "$DD_TRACE_PHP_BIN" ]; then
+    DD_TRACE_PHP_BIN=$(command -v php || true)
+fi
+if [ -z "$DD_TRACE_PHP_BIN" ]; then
+    DD_TRACE_PHP_BIN=$(command -v php7 || true)
+fi
+if [ -z "$DD_TRACE_PHP_BIN" ]; then
+    DD_TRACE_PHP_BIN=$(command -v php5 || true)
 fi
 
-function invoke_php(){
+function invoke_php() {
     # In case of .apk post-install hooks the script has no access the set of exported ENVS.
-    # In case when 1) users import large ini files (e.g. browsercap.ini) and 2) they set memory_limit using an env
+    # When 1) users import large ini files (e.g. browsercap.ini) and 2) they set memory_limit using an env
     # variable we would fail because of the memory limit reached. In order to avoid this we set a sane memory limit
     # that should cause no problem in any machine when our tracer is installed.
-    $DD_TRACE_PHP_BIN -d memory_limit=128M $*
+    # alias invoke_php="$DD_TRACE_PHP_BIN -d memory_limit=128M"
+    $DD_TRACE_PHP_BIN -d memory_limit=128M "$*"
```

Comment:
@morrisonlevi it turned out that escaping the `$*` was enough to have it properly working in all the cases below. So I pretty much restored the old pattern.